profiles/anithasubramanian19.json

```
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/87901316/202974132-6b2abe8b-01c9-4547-b904-9cca271990b9.png",
+    "issueId": 1541,
+    "name": "Anitha Subramanian",
+    "username": "anithasubramanian19"
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```